rec_to_binaries/create_system_time.py

```diff
@@ -0,0 +1,109 @@
+from rec_to_binaries.read_binaries import readTrodesExtractedDataFile, write_trodes_extracted_datafile
+
+import numpy as np
+
+def add_system_time_to_file(continuous_time_file, new_file_name):
+	"""Adds system time to the continuoustime.dat file created by Trodes exporttime command.
+
+    Parameters
+    ----------
+    continuous_time_file : string, filepath to the binary file <date_animal_run>.continuoustime.dat
+    new_file_name: string, filepath to the binary output file <date_animal_run>.continuoustime.dat
+    	which contains the added continuoustime.dat to the 'data' field. Must include the
+    	'contintinuoustime.dat' in the string
+
+    Returns
+    -------
+    Writes out the complete .continuoustime.dat file specified by new_file_name
+
+    """
+
+	cont_time = readTrodesExtractedDataFile(continuous_time_file)
+	cont_time['data'] = create_and_add_system_timepoints(cont_time)
+
+	if 'fields' in cont_time:
+		cont_time['fields'] = '<trodestime uint32><systime int64>'
+	else:
+		cont_time['Fields'] = '<trodestime uint32><systime int64>'
+
+	write_trodes_extracted_datafile(new_file_name, cont_time)
+
+def create_and_add_system_timepoints(cont_time):
+	"""Creates the system time by extrapolating from the 'system_time_at_creation' timestamp
+	as a function of the 'clockrate' and the 'trodestime' containted in 'data'
+
+    Parameters
+    ----------
+    cont_time: dict, a dictionary of various fields and data created by 
+    	readTrodesExtractedDataFile(<filepath_to_continuoustime.dat_file>)
+    -------
+
+
+	Assumptions: 'clockrate' is in Hz; 'system_time_at_creation' is taken to the millisecond
+
+	""" 
+
+	NANOSECONDS_TO_SECONDS = 1e9
+
+	if 'clockrate' in cont_time:
+		nanosec_per_sample = int((1/int(cont_time['clockrate']))* (NANOSECONDS_TO_SECONDS))
+	else:
+		nanosec_per_sample = int((1/int(cont_time['Clockrate']))* (NANOSECONDS_TO_SECONDS))
+
+	sys_time_intervals = np.zeros(np.shape(cont_time['data']), dtype='int64')
```

Comment:
Can you move this lower so that it is next to where `sys_time_intervals` gets used.